backend/manager/modules/bll/src/main/java/org/ovirt/engine/core/bll/storage/disk/image/TransferDiskImageCommand.java

```diff
@@ -1034,6 +1034,12 @@ protected boolean startImageTransferSession() {
         }
 
         Guid imagedTicketId = Guid.newGuid();
+        ImageTransfer updates = new ImageTransfer();
+        updates.setImagedTicketId(imagedTicketId);
+        updates.setVdsId(getVdsId());
+        updates.setProxyUri(getProxyUri() + IMAGES_PATH);
+        updates.setDaemonUri(getImageDaemonUri(getVds().getHostName()) + IMAGES_PATH);
+        updateEntity(updates);
```

Comment:
So now we if we don't have a ticket, it means we don't need to remove it from the daemon and proxy during cleanup?

I think this change makes sense - adding the ticket id before we store is fragile.